.config/dictionaries/project.dic

```diff
@@ -23,6 +23,7 @@ CBOR
 cborg
 CEST
 cfbundle
+cheveron
```

Comment:
Shouldn't it be "chevron"?

Reply:
Not according to he UI kit icons :) Let me check on Figma and change them there!